modelforge/potential/schnet.py

```diff
@@ -162,20 +180,15 @@ def forward(
         # Perform the scatter_add operation
         x_native.scatter_add_(0, idx_i_expanded, x_ij)
 
-        assert torch.equal(x_native, x_custom)
-
         # Update features
         x = self.feature_to_output(x_native)
-        x = x.reshape(batch_size, nr_of_atoms, 128)
+        x = x.reshape(batch_size, nr_of_atoms, self.nr_atom_basis)
         return x
 
 
 class SchNetRepresentation(nn.Module):
     def __init__(
         self,
-        n_atom_basis: int,
-        n_filters: int,
-        n_interactions: int,
         cutoff: float = 5.0,
```

Comment:
It would be good here to specify the units (even without using the unit package, just annotate what units cutoff should be in).  Also I think the data we are loading in is nanometers, so this should be 0.5 (since 5.0 angstrom is used in schnet, right)?

Reply:
that's great catch, thank you!